task.sql

```diff
@@ -9,9 +9,36 @@ CREATE TABLE Countries (
 ) ENGINE=InnoDB;
 
 -- Create a table for caching GeoIP data (Columns: ID, IP Range, CountryID)
+CREATE TABLE GeoIPCache (
+    ID INT,
+    IPRange VARCHAR(50),
+    CountryID INT,
+    FOREIGN KEY (CountryID) REFERENCES Countries(ID) ON DELETE NO ACTION,
+    PRIMARY KEY (ID)
+) ENGINE=Memory;
 
 -- Create a table for storing product descriptions for different countries (Columns: ID, CountryID, ProductID, Description )
+CREATE TABLE ProductDescription (
+    ID INT,
+    Description VARCHAR(50),
+    ProductID INT,
+    FOREIGN KEY (ProductID) REFERENCES Products(ID) ON DELETE NO ACTION,
```

Comment:
Are you sure that we need to use FOREIGN KEY in this situation? We don't have table Products here